### ISSUE TITLE
resource/aws_ecs_service: Upgrading placement_strategy to ordered_placement_strategy still requiring resource replacement in version 2.0.0

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#7787]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
terraform 0.11.11
terraform-aws-provider 2.0.0
```

### Affected Resource(s)

* `aws_ecs_service`

### Terraform Configuration Files

```hcl
# Previous configuration in Terraform AWS Provider 1.X

resource "aws_ecs_service" "example" {
  # ... other configuration ...

  placement_strategy {
    field = "memory"
    type = "binpack"
  }

  placement_strategy {
    field = "attribute:ecs.availability-zone"
    type = "spread"
  }
}

# Attempted configuration update in Terraform AWS Provider 2.0.0
resource "aws_ecs_service" "example" {
  # ... other configuration ...

  ordered_placement_strategy {
    field = "memory"
    type = "binpack"
  }

  ordered_placement_strategy {
    field = "attribute:ecs.availability-zone"
    type = "spread"
  }
}
```

### Expected Behavior

No changes during `terraform plan` except potentially configuration ordering changes to match the configuration within ECS.

### Actual Behavior

```
      ordered_placement_strategy.#:                           "2" => "2"
      ordered_placement_strategy.0.field:                     "memory" => "memory"
      ordered_placement_strategy.0.type:                      "binpack" => "binpack"
      ordered_placement_strategy.1.field:                     "attribute:ecs.availability-zone" => "attribute:ecs.availability-zone"
      ordered_placement_strategy.1.type:                      "spread" => "spread"
      placement_strategy.#:                                   "2" => "0" (forces new resource)
      placement_strategy.2224589570.field:                    "memory" => "" (forces new resource)
      placement_strategy.2224589570.type:                     "binpack" => "" (forces new resource)
      placement_strategy.3619322362.field:                    "attribute:ecs.availability-zone" => "" (forces new resource)
      placement_strategy.3619322362.type:                     "spread" => "" (forces new resource)
```

### Steps to Reproduce

1. `terraform apply` first configuration with Terraform AWS Provider version 1.60.0
2. `terraform apply` second configuration with Terraform AWS Provider version 2.0.0

### Important Factoids

The `placement_strategy` configuration block was removed in version 2.0.0: https://github.com/terraform-providers/terraform-provider-aws/issues/7687

Usually Terraform ignores differences with `Removed` schema attributes, however it seems this is not true when they also have `ForceNew: true` set.

### References

* https://github.com/terraform-providers/terraform-provider-aws/issues/4557#issuecomment-468035357

**Comment by bflad** [comment]
Fix submitted: #7790

**Comment by bflad** [comment]
The fix for this has been merged and will release in version 2.1.0 of the Terraform AWS Provider, likely middle of this week.

**Comment by bflad** [comment]
This has been released in [version 2.1.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#210-march-07-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
